.github/workflows/pypipublish.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+jobs:
+  build-and-publish-python-module:
+    name: Build and publish python module to pypi
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```

Comment:
I feel like there's some inconsistencies of versions here. Like I see ubuntu 16 and 18, and python 3.6 and 3.7. Any way we can standardize those?

Reply:
Hmm yeah I'll use 3.7 here.